### PR TITLE
#112: Implemented mocked login endpoint

### PR DESCRIPTION
**Pull request by bonkey**
- Implemented GET and POST endpoints on mock for /oauth2/authorize
- Added static variable considered in hasMockedAPIEnabled check
- Replaced UI Test login with just “Login” tap
- Replaced all urls in config.json to localhost.charlesproxy.com (for easier debugging with Charles Proxy)
- Removed unused ConfigServiceResponse.json

### CODE REVIEW

**Comment by bonkey** [comment]
Either 👍 and merge on your own, or make fixes on your own and... merge on your own. I have a bus in few hours, tomorrow I'm in a road back and forth almost all day.

**Comment by bonkey** [comment]
👍

**Comment by bonkey** [comment]
Ah, I think UI test is failing after placed order

**Comment by bonkey** [comment]
👍

File: AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
     "atlas-catalog-api": {
-        "url": "http://localhost:9080"
+        "url": "http://localhost.charlesproxy.com:9080"
```
Comment (ahmed-shehata) [inline AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json]: Do you need to remove this?

Comment (bonkey) [inline AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json]: It's adding change, not removing. Don't get the comment.

Comment (ahmed-shehata) [inline AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json]: I mean the charlesproxy part, would this still work without Charles installed?

Comment (bonkey) [inline AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json]: Of course. Until owners of charlesproxy.com pay for the domain and won't do any spoofing.

```
$ host localhost.charlesproxy.com 8.8.8.8
Using domain server:
Name: 8.8.8.8
Address: 8.8.8.8#53
Aliases:

localhost.charlesproxy.com has address 127.0.0.1
```

Comment (bonkey) [inline AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json]: More on this: https://www.charlesproxy.com/documentation/faqs/localhost-traffic-doesnt-appear-in-charles/

**Comment by bonkey** [comment]
:+

**Comment by bonkey** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

**Comment by bonkey** [comment]
Interesting, buddybuild disappeared?

**Pull request merged**